src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -6,10 +6,11 @@
 import java.util.HashSet;
 
 public class WinningLotto {
-    private static final String LOTTO_SIZE_ERROR = "로또 숫자의 입력값은 6개로 이루어져야 합니다.";
 
+    private static final int LOTTO_SIZE = 6;
+    private static final String LOTTO_SIZE_ERROR = "로또 숫자의 입력값은 6개로 이루어져야 합니다.";
     private static final String SPLIT_REGEX = ", ";
-    HashSet<LottoNumber> winningLotto = new HashSet<>();
+    private final HashSet<LottoNumber> winningLotto = new HashSet<>();
```

Comment:
여기도 
```suggestion
    private final Set<LottoNumber> winningLotto = new HashSet<>();
```
타입을 인터페이스로 하는게 좋을거 같아요~